tox.ini

```diff
@@ -30,9 +30,29 @@ ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D
 match-dir = (?!migrations)
 
 [pytest]
-DJANGO_SETTINGS_MODULE = test_settings
-addopts = --cov edx_sysadmin --cov-report term-missing --cov-report xml
-norecursedirs = .* requirements site-packages
+pep8maxlinelength = 119
+DJANGO_SETTINGS_MODULE = lms.envs.test
+addopts = --cov edx_sysadmin --cov-report term-missing --cov-report xml --nomigrations --reuse-db --durations=20
+
+filterwarnings =
+    default
+    ignore:No request passed to the backend, unable to rate-limit:UserWarning
```

Comment:
I think we should have a technical debt story in the backlog to fix these warnings.

Reply:
Although these are thrown from inside edX modules upon usage. But I have created a tech debt issue for it https://github.com/mitodl/edx-sysadmin/issues/21 for a deeper look and any possible fixes.